src/js/protocols/stm32.js

```diff
@@ -8,19 +8,21 @@
 'use strict';
 
 var STM32_protocol = function () {
-    this.baud;
+    this.baud = null;
     this.options = {};
-    this.callback; // ref
-    this.hex; // ref
-    this.verify_hex;
+    this.callback = null;
+    this.hex = null;
+    this.verify_hex = [];
 
-    this.receive_buffer;
+    this.receive_buffer = [];
+
+    this.bytes_to_read = 0;
+    this.read_callback = null;
 
-    this.bytes_to_read = 0; // ref
-    this.read_callback; // ref
+    this.upload_time_start = 0;
+    this.upload_process_alive = false;
 
-    this.upload_time_start;
-    this.upload_process_alive;
+    this.msp_connector = new MSPConnectorImpl();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['MSPConnectorImpl' is not defined.](https://app.codacy.com/gh/emuflight/EmuConfigurator/pullRequest?prid=5626377)